### PR TITLE
Add Help (goto class documentation) to node context menu

### PR DESCRIPTION
**Pull request by ghost**
Will jump directly to class documentation of node class:

docs

### CODE REVIEW

File: editor/scene_tree_dock.cpp [hunk 1]
```diff
@@ -713,6 +713,14 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 				}
 			}
 		} break;
+		case TOOL_GOTO_DOCUMENTATION: {
+
+			List<Node*> selection = editor_selection->get_selected_node_list();
```
Comment (akien-mga) [inline editor/scene_tree_dock.cpp]: Small style issue for `clang-format` here: https://travis-ci.org/godotengine/godot/jobs/308827286

(not that clang-format's style is inherently better in this case, but with automated tools we can't easily make exceptions :))

**Comment by groud** [comment]
Good idea !
However, I would go for something like "Open node's documentation", instead of "Help" which is a little bit evasive.

**Comment by reduz** [comment]
you can go to help from the inspector already... in fact we could make that
button stand out a bit more

**Comment by poke1024** [comment]
Closing this in favor of https://github.com/godotengine/godot/pull/13924

**Pull request closed without merging**